static/index.scss

```diff
@@ -0,0 +1,17 @@
+@import "./styles/colors";
+
+// @font-face {
```

Comment:
Why didn't you add styles (mixins probably?) for fonts?
We have special typography style name for each of the fonts. You may look it up in figma (e.g. bold_20, bold_24 & etc.)

Reply:
Also they need to be placed in separate file (`_typography.scss`)